### PR TITLE
Script to generate messages

### LINKED ISSUE #335: Create script to generate correct messages with optionally different cluster_id, org_id, and account_number
Create script to generate correct messages with optionally different cluster_id, org_id, and account_number

### PR DESCRIPTION
**Pull request by tisnik**
# Description

Script to generate messages

Fixes #335 

## Type of change

- New feature (non-breaking change which adds functionality)
- Documentation update

## Testing steps

Done on CI

## Checklist
* [ ] `make before_commit` passes
* [ ] updated documentation wherever necessary
* [ ] added or modified tests if necessary
* [ ] updated schemas and validators in [insights-data-schemas](https://github.com/RedHatInsights/insights-data-schemas) in case of input/output change

### CODE REVIEW

File: input/gen_messages.py [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+        default="input.json",
+        required=False,
+        help="Specification of input file",
+    )
+
+    parser.add_argument(
+        "-o",
+        "-output",
+        dest="output",
+        default="output{}.json",
```
Comment (juandspy) [inline input/gen_messages.py:90]: Just a nitpick. Maybe we should use `output_{}.json` instead? That way the base filename and index will be more separated.

**Review by juandspy — APPROVED** [review]
super!

File: input/gen_messages.py (hunk 1 above)
Comment (tisnik) [reply input/gen_messages.py:90]: ok

Comment (tisnik) [reply input/gen_messages.py]: fixed

**Review by joselsegura — APPROVED** [review]

**Pull request merged**